packages/frontend/src/components/TaskModal/TaskModal.tsx

```diff
@@ -17,7 +17,6 @@ import GrayButtonFilled from '../GrayButtonFilled/GrayButtonFilled'
 
```

Comment:
remove frontend changes from this branch

Reply:
Done